<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestStyling</class>
 <widget class="QMainWindow" name="TestStyling">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tutorial/TestApp: JKQTPlotter Styling</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="JKQTPlotter" name="plot" native="true"/>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>debug-boxes:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkDebugBoxes">
            <property name="text">
             <string>enabled</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>log scales:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkLogX">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkLogY">
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>grids:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkGrid">
            <property name="text">
             <string>X + Y</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="btnLoad">
            <property name="text">
             <string>Load</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QPushButton" name="btnUpdate">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Update Graph</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="btnSave">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="4">
           <widget class="QPlainTextEdit" name="plainTextEdit">
            <property name="font">
             <font>
              <family>Courier New</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="lineWrapMode">
             <enum>QPlainTextEdit::NoWrap</enum>
            </property>
            <property name="placeholderText">
             <string>enter plotter style INI contents here ...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="4">
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>plotter style:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>863</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>JKQTPlotter</class>
   <extends>QWidget</extends>
   <header>jkqtplotter/jkqtplotter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
